app/controllers/homeworks_controller.rb

```diff
@@ -43,10 +44,7 @@ def update
   # DELETE /homeworks/1.json
   def destroy
     @homework.destroy
-    respond_to do |format|
-      format.html { redirect_to lecture_path(lecture_id), notice: 'Homework was successfully destroyed.' }
-      format.json { head :no_content }
-    end
+    respond_with(@homework)
```

Comment:
i think this should just be `render status: :no_content`

Reply:
I checked the [documentation](https://www.rubydoc.info/gems/responders/3.0.1) for the responders gem and I'm not seeing anything for render status. I was able to destroy, edit, and create homework on my local using this respond_with(@homework) and was redirected back to the homeworks index.